test/nn/simplicial/test_dist2cycle.py

```diff
@@ -0,0 +1,44 @@
+"""Unit tests for Dist2Cycke Model."""
+import numpy as np
+import numpy.linalg as npla
```

Comment:
Remove this line and use np.linalg. everytime you need it.

Best practices are to avoid renaming module elements, because it hides where the elements, functions, etc are coming from when we are reading the code.